kernel/ko_src/kmesh/kmesh_parse_http_1_1.c

```diff
@@ -307,10 +307,13 @@ static bool parse_header(struct bpf_mem_ptr *context)
 					continue;
 				field_value_begin_position = i;
 				field_value_end_position = i;
-				current_state = ST_FIELD_VALUE;
+				if (ch == CR)
```

Comment:
According to the HTTP protocol details in RFC2616, standalone CR or LF is not allowed, so CR is considered as the end of the line.